cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfBotPawn.h

```diff
@@ -12,6 +16,13 @@
 class UCameraComponent;
 class UUrdfRobotComponent;
 
+struct InputAxisBinding
+{
+    FName axis_;
```

Comment:
Use std::string because it's easier to examine in a debugger. Convert using Unreal::toFName() when calling an Unreal function.

Reply:
fixed